### PR TITLE
feat: reference pipeline improvements

### LINKED ISSUE #42: Make the reference pipline take the same input as the ordinary pipeline
At the moment, the reference part of the pipeline requires BAM files and CNV VCF files for each sample. This means that the units files have a different look between normal runs of the pipeline, and runs for generating reference files. This should be unified so that the same format is used for both pipelines, and the reference pipeline should import rules from the ordinary pipline as necessary in order to generate input files for the various reference-specific rules. This would then also mean that reference files can be generated from FASTQ input, and it also ensures that all the input files are compatible with the specific version of the pipeline.

### PR DESCRIPTION
**Pull request by maehler**
This PR aims to streamline the generation of the reference files generated by the reference workflow that is part of the pipeline. Instead of accepting BAM files and VCF files, it utilizes the main pipeline to generate these as input files.

Closes #42

### CODE REVIEW

**Comment by maehler** [comment]
Left to do before I open this for review is some more documentation, and also some more testing to make sure that everything works as it should in a realistic setup.

File: docs/reference_files.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Reference files
+
+## Panel of normals
```
Comment (elleira) [inline docs/reference_files.md:3]:
```suggestion
## Panel of normals for CNV analysis 
```

File: docs/reference_files.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+A pipeline for generating panels of normals for various software is included in Poppy. This leverages the main pipeline when it comes to generating input files for the reference-specific rules, which in turn means that the configuration for the main pipeline in addition to the reference pipeline has to be applied when running it.
+
+As an example, consider the following Snakemake profile (`reference_profile/config.yaml`):
```
Comment (elleira) [inline docs/reference_files.md:7]: Feel stupid but this isn't a real file right? Cause i cant find it :sweat_smile:

Comment (maehler) [reply docs/reference_files.md:7]: Indeed, it is not a real file. It was just meant as an example, and I needed the file name a bit further down. I'll make that a bit more clear.

**Review by elleira — APPROVED** [review]

**Pull request merged**